### PR TITLE
[Compute] Add new command to support upgrading extensions for VMSS

### PR DESCRIPTION
**Pull request by houk-ms**
**Description**  

For feature request #15017. 

This PR provides supports for upgrading all the extensions for all the instances of VMSS to the lastest extension version. The new command added is **az vmss extension upgrade**
```

Command
    az vmss extension upgrade : Upgrade all extensions for all VMSS instances to the latest version.

Arguments
    --no-wait           : Do not wait for the long-running operation to finish.

Resource Id Arguments
    --ids               : One or more resource IDs (space-delimited). It should be a complete
                          resource ID containing all information of 'Resource Id' arguments. You
                          should provide either --ids or other 'Resource Id' arguments.
    --name -n           : Scale set name. You can configure the default using `az configure
                          --defaults vmss=<name>`.
    --resource-group -g : Name of resource group. You can configure the default group using `az
                          configure --defaults group=<name>`.
    --subscription      : Name or ID of subscription. You can configure the default subscription
                          using `az account set -s NAME_OR_ID`.
```

**Testing Guide**  

1. create a vmss
```
vmss create -g {rg} -n {vmss} -l {loc} --authentication-type password --admin-username admin123 --admin-password PasswordPassword1! --image Win2012R2Datacenter
```
2. install an old version extension
```
vmss extension image list-versions -l {loc} -p {publisher} -n {extension}
vmss extension set -g {rg} --vmss-name {vmss} -n {extension} --publisher {publisher} --version {min_version}
```
3. run the new command
```
vmss extension upgrade -g {rg} -n {vmss}
```
4. check the extension version
```
vmss extension list -g {rg} --vmss-name {vmss}
```

### CODE REVIEW

**Comment by yonzhan** [comment]
Compute

File: src/azure-cli/azure/cli/command_modules/vm/_help.py [hunk 1]
```diff
@@ -2428,6 +2428,11 @@
     crafted: true
 """
 
+helps['vmss extension upgrade'] = """
+type: command
+short-summary: Upgrade all extensions for all VMSS instances to the latest version.
+"""
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/vm/_help.py:2434]: Add at least one example for one command?

Comment (houk-ms) [reply src/azure-cli/azure/cli/command_modules/vm/_help.py]: Added.

File: src/azure-cli/azure/cli/command_modules/vm/_help.py [hunk 2]
```diff
@@ -2431,6 +2431,9 @@
 helps['vmss extension upgrade'] = """
 type: command
 short-summary: Upgrade all extensions for all VMSS instances to the latest version.
+examples:
+  - name: Upgrade all extensions to the latest version.
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/vm/_help.py:2435]: Can we specify a certain version?

**Review by qwordy — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/vm/_help.py (hunk 2 above)
Comment (qwordy) [reply src/azure-cli/azure/cli/command_modules/vm/_help.py:2435]: Any comment?

**Pull request merged**